src/components/data-processing/ConfigureEmbedding/ConfigureEmbedding.jsx

```diff
@@ -178,7 +178,6 @@ const ConfigureEmbedding = (props) => {
         experimentId={experimentId}
```

Comment:
based on what I can see, in function `renderContinuousEmbedding` when rendering `ContinuousEmbeddingPlot` you can remove `onUpdate={updatePlotWithChanges}` since `onUpdate` is removed from `ContinuousEmbeddingPlot` component.